xmlStyle.go

```diff
@@ -687,56 +693,50 @@ func (borders *xlsxBorders) Marshal(outputBorderMap map[int]int) (result string,
 // currently I have not checked it for completeness - it does as much
 // as I need.
 type xlsxBorder struct {
-	Left   xlsxLine `xml:"left,omitempty"`
-	Right  xlsxLine `xml:"right,omitempty"`
-	Top    xlsxLine `xml:"top,omitempty"`
-	Bottom xlsxLine `xml:"bottom,omitempty"`
+	Left     xlsxLine `xml:"left,omitempty"`
+	Right    xlsxLine `xml:"right,omitempty"`
+	Top      xlsxLine `xml:"top,omitempty"`
+	Bottom   xlsxLine `xml:"bottom,omitempty"`
+	Diagonal xlsxLine `xml:"diagonal,omitempty"`
```

Comment:
How can a user ever apply it since it's not part of the `Border` struct in `styles.go`? I'd also think it would need to be supported by the `xlsxBorder.Marshal` function.